sdk/servicebus/service-bus/README.md

```diff
@@ -172,31 +172,32 @@ To learn more, please read [Settling Received Messages](https://docs.microsoft.c
 
 To send messages using sessions, you first need to create a session enabled Queue or Subscription. You can do this
 in the Azure portal. Then, use an instance of a `ServiceBusClient` to create a sender using the using
-the [createSender][sbclient_createSender]
+the [createSender][sbclient_createsender]
 function. This gives you a sender which you can use to [send][sender_send] messages.
 
-When sending the message, set the `sessionId` property in the message body to ensure your message
+When sending the message, set the `sessionId` property in the message to ensure your message
 lands in the right session.
 
 ```javascript
 const sender = serviceBusClient.createSender("my-session-queue");
 await sender.send({
   body: "my-message-body",
-  sessionId: "my-session"
+  sessionId: "my-session",
 });
 ```
 
 ### Receive messages from Sessions
 
 To receive messages from sessions, you first need to create a session enabled Queue and send messages
 to it. Then, use an instance of `ServiceBusClient` to create a receiver
-using the [createSessionReceiver][sbclient_createSessionReceiver] function.
+using the [createSessionReceiver][sbclient_createsessionreceiver] function.
 
-Note that you will need to specify the session from which you want to receive messages.
+Specifying a `sessionId` allows you to open a specific session. If the session id is not specified
+then the `SessionReceiver` will get a random unlocked session.
```

Comment:
Why is it important to say "unlocked" here? A user just getting started wouldn't know what a locked/unlocked session is.

I would suggest something like the below:

> Pass the `sessionId` to the `createSessionReceiver` method if you know which session you want to receive messages from. At any given point in time, there can be only one receiver for a given session in the Queue. If no `sessionId` is passed, then the receiver is created for the next available session.

Reply:
I like the way your suggestion reads but I think we should leave it as 'unlocked'. 

Locking is pretty much a critical concept for sessions in general. We could say "next available" but that doesn't actually mean anything in Service Bus and then we'd have to explain what "next available" means which will inevitably mean we have to say something like "unlocked" anyways.

How strongly do you feel about this? ;)

Reply:
> Then, use an instance of `ServiceBusClient` to create a receiver using the [createSessionReceiver][sbclient_createsessionreceiver] function. This locks a session for you as there can be only one receiver for a given session in the Queue.

> Specifying a `sessionId` allows you to receive messages from the corresponding session. If the session id is not specified, then you will get a receiver that takes a lock on the next available session.

Reply:
I've revised it again - take a look. It's not _exactly_ what you wrote but I think I captured the gist.

Reply:
Well, now we are using both "available" and "unlocked", but the problem still remains that somebody not well versed with how session lock works or that session lock exists wouldnt know what this means.

Either we mention explicitly that there can be only one receiver for one session and then use the words "available" and/or "unlocked", or choose to skip this altogether and make no mention of the two words and just say "a random session" or "a session randomly chosen by the service"

Reply:
I know you're probably sick of this but can you take a look again? I realized that maybe we were trying to put in too much meaning into too few words :)

I structured it a bit more and tried to add in some explanation, a chance to establish terms, etc...